helper_scripts/analyze.py

```diff
@@ -51,53 +50,60 @@ def analyze_kafka_dump(directory):
                              order_cost[merchant_id], profit[merchant_id]])
 
     create_inventory_graph(directory, merchant_id_mapping)
+    create_profit_per_minute_graph(directory, merchant_id_mapping)
+    create_revenue_per_minute_graph(directory, merchant_id_mapping)
 
-
-def create_inventory_graph(directory, merchant_id_mapping):
-    """
-    Calculates inventory levels from orders and sales and generates a graph from it.
-    """
-    sales = json.load(open(os.path.join(directory, 'kafka', 'buyOffer')))
-    orders = json.load(open(os.path.join(directory, 'kafka', 'producer')))
-    sales = [sale for sale in sales if sale['http_code'] == 200]
-    for sale in sales:
-        # TODO: ues better conversion; strptime discards timezone
-        sale['timestamp'] = datetime.datetime.strptime(sale['timestamp'], '%Y-%m-%dT%H:%M:%S.%fZ')
-    for order in orders:
+def parse_timestamps(events):
+    for event in events:
         # TODO: ues better conversion; strptime discards timezone
```

Comment:
Typo

Reply:
Are the times currently generated correct? Is the missing time zone support a problem we will see even though we never "move the time stamps" out of the server?

Reply:
Fixed the typo.
The generated times are correct (our local time).
The current solution becomes problematic if the dates have different timezones, which they currently don't have.
The implementation will be more robust if we do not strip out the timezone information.